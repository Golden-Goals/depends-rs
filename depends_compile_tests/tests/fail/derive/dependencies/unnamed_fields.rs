use depends::derives::Dependencies;

#[derive(Dependencies)]
struct Bar(usize, usize, usize);

fn main() {}
