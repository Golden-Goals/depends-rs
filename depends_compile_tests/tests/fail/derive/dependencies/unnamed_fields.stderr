error: Must be a struct with named fields.
 --> tests/fail/derive/dependencies/unnamed_fields.rs:3:10
  |
3 | #[derive(Dependencies)]
  |          ^^^^^^^^^^^^
  |
  = note: this error originates in the derive macro `Dependencies` (in Nightly builds, run with -Z macro-backtrace for more info)
